handbook/engineering/web/index.md

```diff
@@ -118,6 +118,9 @@ We use the backlogs for tracking bugs, small features, and unplanned work. We do
 
 To add an issue, tag it `team/web` to notify the web team PM and put it in the "to triage" column of the board. Unless you're directly asked, only web team members should move issues out of the "to triage" column on the board. A web team member will confirm the issue is web team-related, and then move it to the appropriate column.
 
+#### Tracking Issues
+The web team makes use of [tracking issues](../tracking_issues.md) for tracking progress on the implementation of new features. When starting work on a new and significant feature, the web team should ensure that a tracking issue is created appropriately.
```

Comment:
@umpox @5h1rU thanks for the proposal! Could you add a little (1-2 sentences or phrases) more guidance on what a "new and significant" feature is so that new teammates or others could know when to expect a tracking issue?

Reply:
Have updated!

> The web team should ensure that a tracking issue is created when starting work on features that are expected to take longer than a single iteration to deliver.

I think this best summarises where our potential problems could lie. Additionally, if a single tasks is expected to take longer than a single iteration, then this will encourage us to further break down that work into manageable chunks